Automattic/wp-calypso
# Update branding for Akismet checkout

## Proposed Changes

This PR updates the branding of the Akismet checkout to use the Akismet logo, and a white background

## Testing Instructions

1. Checkout this branch
2. Start your local environment: `yarn start`
3. Head to https://wordpress.com/checkout/business-bundle and choose a site. Ensure the masterbar for WPCOM checkout looks the same as before
![image](https://user-images.githubusercontent.com/65001528/225972716-4439c9cf-9aca-47c6-91ea-429b4180bb19.png)
4. Head to http://calypso.localhost:3000/checkout/jetpack/jetpack_backup_t1_yearly and make sure the jetpack branding looks the same
![image](https://user-images.githubusercontent.com/65001528/225972839-84406149-b8d7-47d7-af86-ad36dcc52566.png)
5. Head to http://calypso.localhost:3000/checkout/akismet/ak_plus_yearly_1 and make sure the Akismet branding of the masterbar looks good.
![image](https://user-images.githubusercontent.com/65001528/225973030-0ad67074-b02f-4caa-9a6a-9b63883e9a8c.png)
 
Note: @ballio2000 was consulted on this design

## Pre-merge Checklist
